curvefs/src/tools/topology/curvefs_build_topology_tool.h

```diff
@@ -101,6 +91,29 @@ class CurvefsTopologyTool {
 
     int TryAnotherMdsAddress();
 
+    void PrintHelp() override {
```

Comment:
If needn't, delete it

Reply:
> If needn't, delete it

this function usde for -example to get help hiw to use this command